### DISCUSSION TITLE
Why is the last link in Breadcrumb disabled?

### DISCUSSION THREAD

**Discussion by ryo-manba** [adobe/react-spectrum#6969] [Q&A]
In the [APG patterns](https://www.w3.org/WAI/ARIA/apg/patterns/breadcrumb/examples/breadcrumb/), the last item (the current page) is an active link.  Why is the last item in [Breadcrumb](https://react-spectrum.adobe.com/react-aria/Breadcrumbs.html) made a disabled link? Given that `<a disabled>` doesn’t exist semantically, wouldn’t it be better to avoid using disabled links?

Scottohara also mentioned that disabled links should be used wisely, carefully, and hopefully rarely in his [blog post](https://www.scottohara.me/blog/2021/05/28/disabled-links.html).

I’d appreciate any thoughts on this. Thanks!

**Comment by snowystinger** [comment]
I'm not entirely sure of the history on this component, @majornista may have more insight. I'm not sure why the current page would be a non-disabled link. It wouldn't go anywhere, as it's the page you're already on. The APG patterns are one way in which to implement something. They are a good guideline, but not a strict rule.

I see our announcement seems to be aligned with what that blog post described, so at the least, according to them, we are not breaking any aria.
![Screenshot 2024-08-29 at 8 21 38 AM](https://github.com/user-attachments/assets/73469046-84d4-4586-b06b-e40c9547106d)
And we don't use a link, only a `role="link"` on a `span` so there is no need to worry about `<a disabled>`
![Screenshot 2024-08-29 at 8 27 35 AM](https://github.com/user-attachments/assets/0a0fcc5f-b3b2-4e9f-8b9b-e8434b5ee69e)

**Reply by ryo-manba** [reply]
By adding `aria-disabled` and `role="link"` to a `span`, aren’t we just creating something like a `<a disabled>`? Since this isn’t a native semantic approach, it feels like a method that isn’t recommended, as mentioned in the blog. What do you think?

**Reply by snowystinger** [reply]
I think that there's more than one way to accomplish a task and that, behaviorally, ours is the same as the recommendation in that blog.

Are you running into a real life issue or is there a bug filed against you for this?

**Reply by ryo-manba** [reply]
There isn't an issue, but I'm wondering if this is the best approach. I don't see why we need to represent that the link is disabled.

If the link could be active based on some interaction, a disabled link might make sense. But in breadcrumbs, the last link will never be active. So, like in the [MDN example](https://developer.mozilla.org/en-US/docs/Web/CSS/Layout_cookbook/Breadcrumb_Navigation), using `<span aria-current="page">` seems like a simpler solution.

I was thinking there might be a specific reason for not following the APG pattern. Could that be the case?

**Reply by snowystinger** [reply]
More than likely it's just that the component was written over six years ago originally and there haven't been any usability issues since then.

But we'll see if anyone remembers more.

**Reply by ryo-manba** [reply]
Thanks! I'll wait for further updates.
